debezium-core/src/main/java/io/debezium/pipeline/spi/OffsetContext.java

```diff
@@ -27,10 +29,10 @@
     /**
      * Implementations load a connector-specific offset context based on the offset values stored in Kafka.
      */
-    interface Loader {
+    interface Loader<O extends OffsetContext> {
```

Comment:
@morozov, added the `O` parameter here, too. This trickled down to quite a few places, but it avoids some casts and raw type warnings.

Reply:
Thanks! This is in line with the subsequent API changes for this feature.